corehq/apps/sso/utils/message_helpers.py

```diff
@@ -19,3 +26,61 @@ def get_success_message_for_trusted_idp(idp, domain_obj):
         domain=domain_obj.name,
         learn_more='#',  # todo include learn more link
     )
+
```

Comment:
i think these utils might be in the wrong utils file. These utilities are targeted for Django's messages framework.
Maybe something like `context_helpers` is more appropriate?